joomla/joomla-cms
# sync admin menu menutype

### Summary of Changes
We use different menutype values for the admin menu. This PR syncs the used name for new installations.

### Testing Instructions
* Install Joomla
* Install an extension

Make sure everything works as before

### Documentation Changes Required
None
